kbase-extension/static/kbase/js/widgets/function_output/kbaseReportView.js

```diff
@@ -805,9 +799,9 @@ define([
             if (self.options.showHTML) {
                 if (self.reportLinks && self.reportLinks.length) {
                     var $ul = $.jqElem('ul');
-                    self.reportLinks.forEach((reportLink) => {
-                        const link_id = StringUtil.uuid();
-                        const $linkItem = $.jqElem('li').append(
+                    self.reportLinks.forEach(function (reportLink) {
+                        var link_id = new UUID(4).format();
+                        var $linkItem = $.jqElem('li').append(
```

Comment:
why the switch to `var`?

Reply:
I'm gonna fix/rewrite/unravel this widget in the next PR. I only made the one-liner change here because it was broken due to the merge and would have broken existing app cells.